gulpfile.js

```diff
@@ -82,7 +108,7 @@ gulp.task('sass', function() {
   }))
   .pipe(sourcemaps.write('./'))
   .pipe(gulp.dest(cssOutput))
-  .pipe(gulp.dest(themePath + projectName + '/css'))
+  .pipe(gulp.dest(css_destination))
```

Comment:
We should probably have similar variables names here... for example maybe pl_css_destination and theme_css_destination

or maybe *_output, less long to write